### PR TITLE
Move all middleware/app config to middleware

### PR DESCRIPTION
**Pull request by ErisDS**
This PR should not result in any functional changes. 

I've just moved some middleware / express setup that was in the server to the middleware layer... it's a small step in the right direction towards being able to easily see all the moving parts for the express app initialisation so that we can then cleanup our middleware something like is described in #4172.

no issue
- move some middleware setup that was in the server index into the middleware index

### CODE REVIEW

File: core/server/middleware/index.js [hunk 1]
```diff
@@ -50,9 +54,31 @@ middleware = {
...
+    blogApp.set('view engine', 'hbs');
+
+    // Create a hbs instance for admin and init view engine
+    adminApp.set('view engine', 'hbs');
+    adminApp.engine('hbs', adminHbs.express3({}));
+
+    // Load helpers
+    helpers.loadCoreHelpers(adminHbs);
+
+    // end of moved code
```
Comment (kirrg001) [inline core/server/middleware/index.js]: you wanne keep "end of moved code" comment?

Comment (ErisDS) [inline core/server/middleware/index.js]: Nah, that was an artefact of what I intended to do next & was too much for one small coding session 😉 

Removed it now.

File: core/client [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 61fe83dff375a145d5272a5382aa2c72bd994830
+Subproject commit 4b3bc50b04cd28f303e6981e2f027c3eef24e592
```
Comment (kirrg001) [inline core/client]: Small question in general:
I think in general we should try to avoid updating `core/client` or?

Comment (ErisDS) [inline core/client]: This was an error, fixed now

**Comment by kirrg001** [comment]
LGTM 👍

**Pull request merged**